tests/config/nf-test.config

```diff
@@ -14,6 +14,7 @@ process {
 profiles {
     singularity {
         singularity.enabled = true
+        singularity.registry = 'quay.io'
```

Comment:
Do these need to be in profile scope? I feel like the registry info should live in the config top-level, for anyone running with their own custom configurations.

Reply:
Good point. Think for `nf-test` the idea is to always specify some sort of `--profile` when testing because you can't pass external configs as far as I can tell. Maybe @sateeshperi can confirm. Either way, I don't think there is any harm in adding it to the top-level like we do in pipelines.

Reply:
Done https://github.com/nf-core/modules/pull/3499/commits/ea8564e9618b7c725bd3942efe84dda0e29854d3